### DISCUSSION TITLE
Can't view details of reports on mobile

### DISCUSSION THREAD

**Discussion by gerdusx** [solodit/feedback#8] [New Feature Requests]
I cant view the report details if the top summary part is too big. Maybe put the summary part also in the scroll section?

See screenshot below

![Screenshot_20230608-184614](https://github.com/solodit/feedback/assets/17019506/32fccd7e-e4e6-4834-babb-c6ed2389abff)

**Comment by wang1252** [comment]
We are now handling mobile responsive UI and plan to release within this weekend.
Thank you for your feedback.

**Comment by hans-solodit** [comment]
@gerdusx we have completed the mobile responsiveness work, and we believe the issue should be resolved. 
We appreciate your feedback.

**Reply by gerdusx** [reply]
@hans-solodit thats perfect! Thanks

**Comment by hans-solodit** [comment]
This issue has been resolved.
